randomizer/config/ResidualFetchHookRandomized/random.json

```diff
@@ -0,0 +1,12 @@
+{
+    "env": "FetchPushDefault-v0",
```

Comment:
Change the name to be correct for all environments.

Reply:
okay! I did.